main.cpp

```diff
@@ -1,48 +1,40 @@
 #include "Character.h"
 #include <iostream>
 
-int main(const int argc, const char** argv) {
+int main (const int argc, const char* argv[])
+{
+	if (argc == 3)
+	{
+		Character player1 = Character::parseUnit(argv[1]);
+		Character player2 = Character::parseUnit(argv[2]);
+		
```

Comment:
feladatkiiras alapjan ennek van lehetosege exceptiont dobni. Ha nem dob exceptiont, akkor az a baj, ha dob, akkor meg nincs lekezelve itt.

Reply:
@hegyhati első exception, amire figyeltünk az az argumentumok száma (3: compile során generált a.out binary, első unit elérési útvonala, második unit elérési útvonala), ami még a legelső feladatból származott. Itt, amennyiben ez nem teljesül parsing előtt elszáll az alábbi hibával, ami a 37. sorban található:
`throw std::runtime_error("Incorrect number of arguments!")`

a második exception az a `parseUnit()` függvényben (Character.cpp) valósul meg, ahol az argumentumból kapott fájlok bekerülnek egy file streambe. Ha a file stream megfelelő, akkor folytatódik a parsing, ha nem, akkor szintén elszáll egy hibával, az alábbi logikát követve:
```
Character Character::parseUnit(std::string fileName){
	std::ifstream sourceFile(fileName);
	if (sourceFile.good())
	{
		...
	}
	else
	{
		throw std::runtime_error("File not found!");
	}
}
```

Reply:
Itt az utobbirol van szo. Az oke, hogy a parseUnit-ban dobtok, csak azt itt try-catch-elni kellene :-)